format/private/format.sh

```diff
@@ -145,7 +145,7 @@ function ls-files {
       # Note: https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/lib/linguist/languages.yml#L7767-L7882
       # has a giant list of patterns. We arbitrarily choose some "common" ones.
       'XML') patterns=('*.xml' '*.xsd') ;;
-
+      'Gherkin') patterns=('*.feature') ;;
```

Comment:
could you include `.story` as well per https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/lib/linguist/languages.yml#L2371-L2381 ?